modules/ximgproc/src/sparse_match_interpolators.cpp

```diff
@@ -36,8 +36,10 @@
 
 #include "precomp.hpp"
 #include "opencv2/ximgproc/sparse_match_interpolator.hpp"
+#include "opencv2/video.hpp"
```

Comment:
Module's dependency list should know that.
Please avoid indirect dependencies.

Reply:
video.hpp is required for the application of the VariationalRefinement method. However even if I add opencv_video to the ocv_define_module definitions. Is the a better way avoid this indirect dependency?